pyomo/contrib/pynumero/src/ma27Interface.cpp

```diff
@@ -0,0 +1,285 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+
+// This would normally be in a header file, but as we do not need one,
+// we will explicitly include it here.
+#if defined(_WIN32) || defined(_WIN64)
+#  if defined(BUILDING_PYNUMERO_ASL)
```

Comment:
I think this should be `BUILDING_PYNUMERO_MA27`, right?

Reply:
Yes.  Done.

Reply:
Thanks!